README.md

```diff
@@ -96,6 +96,93 @@ from server import app as application
 
 > Alternatively these config values can be manually set in `config.py`.
 
+## Persistent SQLite Storage to S3 & Docker
```

Comment:
Looks like some of the header numberings might be a bit off and it would make sense to put `Persistent SQLite Storage to S3 & Docker` under `Docker` maybe.

This file is getting pretty big, I might look at creating a `/docs/` directory that we can relatively link to for more detailed documentation.